README.md

```diff
@@ -31,6 +31,15 @@ struct Cli {
 }
 ```
 
+## Limitations
+
+[Argument methods][argument_methods] support with required clap(flatten) - see clap [derive reference][derive_reference]
+* #27 example with `#[clap(global)]` is not supported
+* [clap-rs/clap/issues/3269](https://github.com/clap-rs/clap/issues/3269)
+
+[argument_methods]: https://docs.rs/clap/latest/clap/struct.Arg.html#implementations
+[derive_reference]: https://github.com/clap-rs/clap/blob/master/examples/derive_ref/README.md
```

Comment:
This seems a bit to specialized to be included in the top-level readme and seems more appropriate for either the reference or the tutorial (I lean towards the derive reference).

Reply:
Thanks done.